Makefile

```diff
@@ -24,15 +24,15 @@ push:
 
 	docker push digitalmarketplace/base:${BUILD_VERSION}
 	docker push digitalmarketplace/base:${BUILD_VERSION}-${BUILD_DATE}
-	docker push digitalmarketplace/base:latest
+	if [ -z $$NOT_LATEST ]; then docker push digitalmarketplace/base:latest; fi
```

Comment:
I feel like this would be more explicit if we could use the Makefile as `make PUSH_LATEST=false` i.e. if we had a Make variable called `PUSH_LATEST` that defaulted to `true`

Reply:
ahbut then y'know, what do you do when someone puts it in as e.g. `PUSH_LATEST=1` (which is my first instinct)?